packages/core/src/codewhisperer/views/lineAnnotationController.ts

```diff
@@ -384,14 +404,13 @@ export class LineAnnotationController implements vscode.Disposable {
             // special case
             // Endstate is meaningless and doesnt need to be rendered
             this.clear()
-            await this.markTutorialDone()
+            await this.dismissTutorial()
             return
         } else if (decorationOptions.renderOptions?.after?.contentText === new TryMoreExState().text()) {
             // special case
             // case 3 exit criteria is to fade away in 30s
             setTimeout(async () => {
-                await this.markTutorialDone()
```

Comment:
this function should only be called when user dismiss the tutorial because the entire `lineAnnotationController` will simply skip logic afterward